content/04.model-complexity.bootstrapping.md

```diff
@@ -0,0 +1,10 @@
+## Bootstrapping {.page_break_before}
+
+Bootstrap or resampling computation is a powerful statistical technique that can be used for estimating population values from datasets of limited sample size [@doi:10.1080/01621459.1997.10474007].
+The technique utilizes random sampling of data points from a dataset of limited sample size with replacement to approximate a larger population and estimate various population statistics (e.g. mean).
+Subsequent iterations of resampling generates a distribution of the statistical value (mean) which minimizes the error of the estimate.
+Bootstrap based techniques are used in conjunction with various learning methods to find the most informative models given a specific dataset [(e.g. bootstrap aggregating or bagging used in random forests)@doi:10.1023/A:1010933404324, @doi:10.1198/0003130043277, (bootstrap in neural networks)@doi:10.1016/S0925-2312(01)00650-6, (regression models)@doi:10.1016/j.neucom.2004.11.017, @doi:10.1002/sim.4780111607].
```

Comment:
I don't know how to format citations like `(e.g. bootstrapping  [@doi:xxxxx])` in Manubot to be honest. I had something similar come up in #63 and I don't know what the right thing to do is.

Reply:
hmm.. good point.. I had not checked how they would render ....

I wonder if it would work to separate them out like (e.g. bootstrapping[@doi:xxx], XYZ [@doi:xxx])